routersploit/interpreter.py

```diff
@@ -159,6 +159,7 @@ class RoutersploitInterpreter(BaseInterpreter):
     help                        Print this help menu
     use <module>                Select a module for usage
     exec <shell command> <args> Execute a command in a shell
+    search <search term>        Search for appropriate module
```

Comment:
Please also add support for `search` command to the completer, you can do that by modifying `suggested_commands()` method, such modification will cause test to fail so you will have to fix them accordingly.